python/beeai_framework/tools/search/base.py

```diff
@@ -11,7 +11,7 @@
 
 class SearchToolResult(BaseModel):
     title: str
-    description: str
+    content: str
```

Comment:
This would be a breaking change. I think that you don't need to inherit from that class.

Reply:
I would create a custom class for the RAG Tool outputs (`RAGToolOutput`).

Reply:
As this is only the retrieval part of RAG, and I believe it's better suited as a search tool, reverting this change